quasar.conf.js

```diff
@@ -12,6 +12,10 @@ module.exports = function (ctx) {
       'app.styl'
     ],
 
+    plugins: [
+      'vuelidate'
```

Comment:
You're only using this package to validate email and it's not connected properly or added to the project dependencies.

Look at the file `src/server/authentication.js` where the `validator` package is used to check if an email is valid. Do the same here. `validator` is included in our project dependencies, so let's switch to using that instead of `vuelidate`.